<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateActe</class>
 <widget class="QDialog" name="updateActe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update Acte</string>
  </property>
  <widget class="QComboBox" name="nrMatricolBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data depunerii:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Denumire act:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="denumireActBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Serie&amp;Nr. act:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serieActEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okInsert">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelInsert">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd/MMM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>9</month>
     <day>10</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr. matricol:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
